tests/end2end_tests/test_create_filtered_tokenized_dataset.py

```diff
@@ -0,0 +1,83 @@
+import tempfile
+from pathlib import Path
+from typing import Callable
+
+import pytest
+
+from modalities.api import FileExistencePolicy, create_filtered_tokenized_dataset
+from modalities.dataloader.dataset import PackedMemMapDatasetBase
+
+
+@pytest.mark.parametrize(
+    "input_data_path, file_existence_policy, filter_routine, expected_exception",
+    [
+        (
+            Path("/raid/s3/opengptx/max_lue/repositories/modalities/tests/end2end_tests/lorem_ipsum.pbin"),
+            FileExistencePolicy.ERROR,
+            lambda x: True,  # take every sample
+            False,
+        ),
+        (
+            Path("/raid/s3/opengptx/max_lue/repositories/modalities/tests/end2end_tests/lorem_ipsum.pbin"),
+            FileExistencePolicy.ERROR,
+            lambda x: False,  # take no sample
+            True,
+        ),
+        (
+            Path("/raid/s3/opengptx/max_lue/repositories/modalities/tests/end2end_tests/lorem_ipsum.pbin"),
```

Comment:
```suggestion
            Path("tests/end2end_tests/lorem_ipsum.pbin"),
```
This makes the single-GPU tests fail. They run through with the suggested change.